setup.json

```diff
@@ -33,10 +33,7 @@
             "sphinxcontrib-details-directive"
         ],
         "pre-commit": [
-            "pre-commit==1.17.0",
-            "yapf==0.28.0",
-            "prospector==1.1.7",
-      	    "pylint==2.3.1"
+            "pre-commit==2.10.1"
```

Comment:
Ah, I see you opted to leave out `black` and `pylint` completely. That's also fine.